lib/normalizers/itemChildrenAreAlwaysElements.js

```diff
@@ -15,27 +52,26 @@ export function itemChildrenAreAlwaysElements(
 
     editor.normalizeNode = (entry: NodeEntry): void => {
         const [node, nodePath] = entry;
-        let parentNodePath;
 
-        try {
-            parentNodePath = Path.parent(nodePath);
-        } catch (e) {
-            // has no parent (ie. editor)
+        if (!isItem(options)(node) || node.children.length === 0) {
             normalizeNode(entry);
 
             return;
         }
 
-        const parentNode = Node.get(editor, parentNodePath);
-
-        if (!Element.isElement(node) && isItem(options)(parentNode)) {
-            const wrapperItem = {
-                type: options.typeDefault,
-                children: [],
-            };
+        if (node.children.some(child => !Element.isElement(child))) {
```

Comment:
What do you think about extracting this condition to `hasTexts`? #CognitiveEase 🍼